packages/cli/src/cli/index.ts

```diff
@@ -302,16 +338,17 @@ const main = async () => {
     .option("--zksync", "Deploy on ZKSync")
     .option("-k, --key <key>", "API key to authorize usage")
     .action(async (options) => {
-      let apiSecretKey = "";
+      let authToken = "";
       // If no key is passed in, prompt the user to login. If it is passed in, use it.
       if (!options.key) {
-        apiSecretKey = await loginUser(cache);
+        authToken = await loginUser(credsConfigPath, cliWalletPath);
       } else {
-        await validateKey(options.key);
-        apiSecretKey = options.key;
+        // Don't need to validate anymore? Also, should probably rely on the underlying service to throw that error.
+        // await validateToken(options.key);
+        authToken = options.key;
```

Comment:
we'll want to support either auth token *or* secret key (secret key still makes sense for CI I think)